tf2onnx/onnx_opset/generator.py

```diff
@@ -88,8 +88,8 @@ def version_9(cls, ctx, node, **kwargs):
         if node.inputs[0].is_const():
             cls.version_1(ctx, node, **kwargs)
         else:
-            seed = node.get_attr("seed")
-            node.set_attr("seed", float(seed.f))
+            seed = node.get_attr("seed2")
```

Comment:
Do we need to also update this for version_1(line 63)?

Reply:
thanks @fatcat-z you're quite right, ill update version_1 as well.

Reply:
I'm a little confused: is this behavior (using seed2 instead of seed) same among different TF versions? Did it change after one of tf version?

Reply:
As far as I've seen and tested, version_1 behaviour is also relevant for TF version.2 
when the random generator dim/shape are constant and not derived from the data 
batch size or any other dependent size.

I'm checking with TF V1 whether it was changed.